lib/auth/webauthncli/register.go

```diff
@@ -178,7 +178,7 @@ func parseU2FRegistrationResponse(resp []byte) (*u2fRegistrationResponse, error)
 	// above.
 	buf := resp
 	if buf[0] != 0x05 {
```

Comment:
Don't you have a constant now you can use in place of the 0x05?

Reply:
I don't want to pull from the `mocku2f` package, nor it seems worth to link both just for this.